content/Watsonx/Assistant/104.md

```diff
@@ -13,8 +14,7 @@ improving your assistant’s performance.
 ## Prerequisites
 
 Complete this section with the assistant you used in the 103 section **or** your own assistant. If you
-want to use your own assistant, you must [upload](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-admin-backup-restore#backup-restore-import) the following [Action skill](https://raw.githubusercontent.com/CloudPak-Outcomes/Watson-Asst-Lab/main/action-skills/begin-104-action-skill.json) which reflects the
-skill after completing section 103.
+want to use your own assistant, you must [upload](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-admin-backup-restore#backup-restore-import) the following [Action skill](https://raw.githubusercontent.com/CloudPak-Outcomes/Watson-Asst-Lab/main/action-skills/begin-104-action-skill.json) which reflects the skill after completing section 103, but note that this will overwrite your existing actions.
 
 ## Introducing Analytics
 
```

Comment:
Include event numbers 1 and 2 like.  Analyze(1) and Draft(2)

Reply:
added